src/editor-plugin/plugin.js

```diff
@@ -60,4 +61,43 @@ const EditorPlugin = () => {
 	);
 }
 
+function onActivityPubPreview() {
+	const previewLink = select( 'core/editor' ).getEditedPostPreviewLink();
+	const fediversePreviewLink = addQueryArgs( previewLink, { activitypub: 'true' } );
+
+	window.open( fediversePreviewLink, '_blank' );
+}
+
+const EditorPreview = () => {
+	const el = React.createElement;
```

Comment:
While directly calling `createElement` will work, it's not a typical way of working. If we look at the [devdocs page for @wordpress/primitives](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-primitives/), I see a different way. Although maybe the `text` node makes it hard to do that?

Reply:
the question is, if this is really the right way... I experimented a bit, because I wanted to know if it is possible to load a text into an SVG container. Maybe it is better practice to load the SVG instead: https://symbol.fediverse.info/

Reply:
what about this instead:

<img width="303" alt="Screenshot 2024-10-24 at 10 43 05" src="https://github.com/user-attachments/assets/26078b27-8784-490e-b2ca-c0706a054133">

Reply:
I like keeping it textual 👍